src/connection.rs

```diff
@@ -1,8 +1,8 @@
 //! A message-oriented API wrapping the underlying QUIC library (`quinn`).
 
 use crate::{
-    config::{RetryConfig, SERVER_NAME},
-    error::{ConnectionError, RecvError, RpcError, SendError, StreamError},
+    config::SERVER_NAME,
+    error::{ConnectionError, RecvError, RpcError, SendError, SerializationError, StreamError},
```

Comment:
Seems that this `SerializationError` was removed by @bochaco, not sure why you introduced it again here.